### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by sandy-marrero** [uprm-inso4117-2023-2024-s2/semester-project-uprmarketplace#2] [Ideas]
We all might have different expectations of what a marketplace application is and the features it should have. Let's use this discussion to gather everyone's feedback and get a clear idea of what app we are going make and what features we will work on.

**Comment by YeranConcepcion** [comment]
-Signup/Login
-Profile Creation
-Browse Products
-Reviews
-Upload Product
-Remove product
-

**Comment by BreannaGSegarra** [comment]
From the class on 2/2/2024:
1. Web application.
2. Signup and login.
3. Profile creation.
4. My own storefront.
5. Browse offer.
6. Review seller/buyer.
7. Filter browsing.
8. Search buyers.
9. Layout of interface.

**Comment by MarkoSchuetz** [comment]
These features still need detail. For example, profile creation: what does it mean? What actions need to be supported. For example, requiring some mandatory information, soliciting optional information, validation of some inputs, providing help texts on hover,...What does it mean for the feature to be acceptable?
